staging/src/k8s.io/kubectl/pkg/cmd/debug/profiles.go

```diff
@@ -49,11 +62,195 @@ func profileLegacy(pod *corev1.Pod, containerName string, target runtime.Object)
 			})
 		}
 
-		pod.Spec.HostIPC = true
-		pod.Spec.HostNetwork = true
-		pod.Spec.HostPID = true
+		setHostNamespace(pod, true)
```

Comment:
well - it does get called in quite a number of places so I prefer to keep it. But maybe we should give this a better name?

Reply:
This function includes 3 lines. Since in Go readability has top priority, it would be better just explicitly set in the place instead calling this function.